README.md

```diff
@@ -115,8 +115,8 @@ deleteVDB loadFromProperties: true, credentialId: '<Input DCT Key 1.abc123456789
 
 ## <a id="links"></a>Links
 
-*   [Delphix Plugin Jenkins Wiki Page](https://wiki.jenkins.io/display/JENKINS/Delphix+Plugin)
-*   [Jenkins Plugin Delphix Page](https://plugins.jenkins.io/delphix)
+*   [Delphix Ecosystem Documentation](https://ecosystem.delphix.com/docs/main/jenkins)
+*   [Jenkins Wiki Page](https://plugins.jenkins.io/delphix)
```

Comment:
Instead of Wiki, can we use Jenkins Plugin Page or Delphix Plugin for Jenkins Page?

Reply:
Good catch. Wiki is something else. 

How does `Jenkins Plugin Site Page` sound? 

"plugin site" seems to be the standard here: https://www.jenkins.io/doc/developer/publishing/documentation/

Reply:
"plugin site" goes better with Jenkins terminology

Reply:
@ankit-patil-hubs Is that in support of:
- Jenkins Plugin Site Page
or
- Plugin Site

Reply:
Jenkins Plugin Site Page

Reply:
Updated